x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_event_log_list_table.tsx

```diff
@@ -0,0 +1,616 @@
+      const result = await loadGlobalConnectorExecutionLogAggregations({
+        sort: formattedSort as LoadGlobalConnectorExecutionLogAggregationsProps['sort'],
+        outcomeFilter: filter,
+        message: searchText,
+        dateStart: getParsedDate(dateStart),
+        dateEnd: getParsedDate(dateEnd),
+        page: pagination.pageIndex,
+        perPage: pagination.pageSize,
+        namespaces,
+      });
+      setLogs(result.data);
+      setPagination({
+        ...pagination,
+        totalItemCount: Math.min(result.total, MAX_RESULTS),
+      });
+      setActualTotalItemCount(result.total);
+    } catch (e) {
+      notifications.toasts.addDanger({
+        title: API_FAILED_MESSAGE,
+        text: e.body?.message ?? e,
+      });
+    }
+    setIsLoading(false);
+  };
+
+  const onChangeItemsPerPage = useCallback(
+    (pageSize: number) => {
+      setPagination((prevPagination) => ({
+        ...prevPagination,
+        pageIndex: 0,
+        pageSize,
+      }));
+    },
+    [setPagination]
+  );
+
+  const onChangePage = useCallback(
+    (pageIndex: number) => {
+      setPagination((prevPagination) => ({
+        ...prevPagination,
+        pageIndex,
+      }));
+    },
+    [setPagination]
+  );
+
+  const onTimeChange = useCallback(
+    ({ start, end, isInvalid }: OnTimeChangeProps) => {
+      if (isInvalid) {
+        return;
+      }
+      setDateStart(start);
+      setDateEnd(end);
+    },
+    [setDateStart, setDateEnd]
+  );
+
+  const onRefresh = () => {
+    setInternalRefreshToken(Date.now());
+    loadEventLogs();
+  };
+
+  const onFilterChange = useCallback(
+    (newFilter: string[]) => {
+      setPagination((prevPagination) => ({
+        ...prevPagination,
+        pageIndex: 0,
+      }));
+      setFilter(newFilter);
+    },
+    [setPagination, setFilter]
+  );
+
+  const onSearchChange = useCallback(
+    (e) => {
+      if (e.target.value === '') {
+        setSearchText('');
+      }
+      setSearch(e.target.value);
+    },
+    [setSearchText, setSearch]
+  );
+
+  const onKeyUp = useCallback(
+    (e) => {
+      if (e.key === 'Enter') {
+        setSearchText(search);
+      }
+    },
+    [search, setSearchText]
+  );
+
+  const onShowAllSpacesChange = useCallback(() => {
+    setShowFromAllSpaces((prev) => !prev);
+    const nextShowFromAllSpaces = !showFromAllSpaces;
+
+    if (nextShowFromAllSpaces && !visibleColumns.includes('space_ids')) {
+      const connectorNameIndex = visibleColumns.findIndex((c) => c === 'connector_name');
+      const newVisibleColumns = [...visibleColumns];
+      newVisibleColumns.splice(connectorNameIndex + 1, 0, 'space_ids');
+      setVisibleColumns(newVisibleColumns);
+    } else if (!nextShowFromAllSpaces && visibleColumns.includes('space_ids')) {
+      setVisibleColumns(visibleColumns.filter((c) => c !== 'space_ids'));
+    }
+  }, [setShowFromAllSpaces, showFromAllSpaces, visibleColumns]);
+
+  const columns: EuiDataGridColumn[] = useMemo(
```

Comment:
Did we want to make the connector name the first column like how we have it for global rule event logs?

![image](https://user-images.githubusercontent.com/74562234/213539186-a9852dd6-f1dd-4127-a505-18f8a47a086a.png)

Reply:
Also did we want to have the connectors clickable? I think it might be helpful in the cross-space scenario to navigate the user to the corresponding connector page

Reply:
Oh that is a good idea! I am wondering if this maybe this could be a follow on issue?